utils/env.go

```diff
@@ -34,6 +34,7 @@ type Env struct {
 	ArgoToken                  string `mapstructure:"ARGO_TOKEN"`
 	ArgoServerURL              string `mapstructure:"ARGO_SERVER_URL"`
 	WorkerRebuildWorkflowFiles string `mapstructure:"WORKER_REBUILD_WORKFLOW_FILES"`
+	StorageRebuildWorkflowFiles string `mapstructure:"WORKER_REBUILD_WORKFLOW_FILES"`
```

Comment:
`mapstructure:"WORKER_REBUILD_WORKFLOW_FILES"` should be `mapstructure:"STORAGE_REBUILD_WORKFLOW_FILES"`

Reply:
changed